src/main/java/thePackmaster/powers/rimworldpack/BurningPassionPower.java

```diff
@@ -0,0 +1,66 @@
+package thePackmaster.powers.rimworldpack;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.BetterOnApplyPowerPower;
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.OnReceivePowerPower;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class BurningPassionPower extends AbstractPackmasterPower implements CloneablePowerInterface, BetterOnApplyPowerPower, OnReceivePowerPower {
+    public static final String POWER_ID = makeID(BurningPassionPower.class.getSimpleName());
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public BurningPassionPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+    }
+
+    @Override
+    public void updateDescription()
+    {
+        description = DESCRIPTIONS[0] + amount + DESCRIPTIONS[1];
+    }
+
+    @Override
+    public AbstractPower makeCopy()
+    {
+        return new BurningPassionPower(owner, amount);
+    }
+
+    @Override
+    public boolean betterOnApplyPower(AbstractPower abstractPower, AbstractCreature abstractCreature, AbstractCreature abstractCreature1)
+    {
+        return true;
+    }
+
+    @Override
+    public boolean onReceivePower(AbstractPower power, AbstractCreature target, AbstractCreature source) {
+        if(power.amount != 0 && target == owner && power.type == PowerType.BUFF)
+        {
+            flash();
+            power.amount *= amount;
+            power.updateDescription();
+        }
+        for (AbstractCard card: AbstractDungeon.player.hand.group)
```

Comment:
I'm surprised this is needed, but even if it isn't needed, I guess it shouldn't do any harm.